torch/csrc/jit/graph_executor.h

```diff
@@ -1,6 +1,8 @@
 #pragma once
 
+#include <torch/csrc/autograd/function.h>
```

Comment:
How much of this is actually changed, versus just code motion? (Can we split this off into a diff before this in the stack?)

Reply:
Furthermore, I would not expect our executors to change at all with this patch. Why did they need to change? This part of the PR is very hard to review because it mixes a ton of code motion with some changes.

Reply:
yea, it'll be split out in any later (non-WIP) diffs